core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/internal/UserStoreMgtDataHolder.java

```diff
@@ -47,4 +52,40 @@ public CryptoService getCryptoService() {
         return cryptoService;
     }
 
+    /**
+     * Set each HashProviderFactory instance in a Map.
```

Comment:
`Set HashProviderFactory to the HashProviderFactory collection`

Reply:
Resolved.